src/plugins/ear-training/ear-training.less

```diff
@@ -1,5 +1,10 @@
 .EarTraining {}
 
+.EarTraining-soundPlayer {
+  .u-width-50;
```

Comment:
I'm not sure if we use a utility class as a mixin anywhere else in the code, I'm kinda OK with it (although we also could have just `width: 50%;` here).

Reply:
EDIT: Little bit of history: Those `u-width-xy` classes exist primarily to be applied dynamically when plugins render (according what the user has set).

Reply:
:+1: